Documentation/operations/upgrade.rst

```diff
@@ -463,36 +463,48 @@ As the :term:`ConfigMap` is successfully upgraded we can start upgrading Cilium
 Migrating from kvstore-backed identities to Kubernetes CRD-backed identities
 ----------------------------------------------------------------------------
 
-Beginning with cilium 1.6, Kubernetes CRD-backed security identities can be
-used for smaller clusters. Along with other changes in 1.6 this allows
+Beginning with Cilium 1.6, Kubernetes CRD-backed security identities can be
+used for smaller clusters. Along with other changes in 1.6, this allows
 kvstore-free operation if desired. It is possible to migrate identities from an
 existing kvstore deployment to CRD-backed identities. This minimizes
 disruptions to traffic as the update rolls out through the cluster.
 
-Affected versions
-~~~~~~~~~~~~~~~~~
-
-* Cilium 1.6 deployments using kvstore-backend identities
-
-Mitigation
-~~~~~~~~~~
+Migration
+~~~~~~~~~
 
-When identities change, existing connections can be disrupted while cilium
+When identities change, existing connections can be disrupted while Cilium
 initializes and synchronizes with the shared identity store. The disruption
 occurs when new numeric identities are used for existing pods on some instances
 and others are used on others. When converting to CRD-backed identities, it is
 possible to pre-allocate CRD identities so that the numeric identities match
-those in the kvstore. This allows new and old cilium instances in the rollout
+those in the kvstore. This allows new and old Cilium instances in the rollout
 to agree.
 
-The steps below show an example of such a migration. It is safe to re-run the
-command if desired. It will identify already allocated identities or ones that
+There are two ways to achieve this: you can either run a one-off ``cilium preflight migrate-identity`` script
+which will perform a point-in-time copy of all identities from the kvstore to CRDs (added in Cilium 1.6), or use the "Double Write" identity
+allocation mode which will have Cilium manage identities in both the kvstore and CRD at the same time for a seamless migration (added in Cilium 1.17).
```

Comment:
Is `added in Cilium 1.17` accurate? I think that these changes will land in a patch release of 1.16 before they land in 1.17.

Reply:
I am not sure tbh but Marco suggested 1.17 here: https://github.com/cilium/cilium/pull/31920#discussion_r1660993202

There are still a couple of unresolved issues in this PR so I am not sure how long all of it will take.

Reply:
> I think that these changes will land in a patch release of 1.16 before they land in 1.17.

Hmm, not sure I'm following. This PR will be part of v1.17, as it missed the v1.16 feature freeze some weeks ago, and being a new feature it doesn't qualify for backport to stable branches.

Reply:
Ah I see, I got confused. Thanks for clarifying!